benefits/core/models.py

```diff
@@ -14,14 +16,29 @@ class PemData(models.Model):
     """API Certificate or Key in PEM format."""
 
     id = models.AutoField(primary_key=True)
-    # The data in utf-8 encoded PEM text format
-    text = models.TextField()
     # Human description of the PEM data
     label = models.TextField()
+    # The data in utf-8 encoded PEM text format
+    text = models.TextField(null=True)
+    # Public URL hosting the utf-8 encoded PEM text
+    remote_url = models.TextField(null=True)
+    # Flag if this PemData has been loaded from remote
+    remote_loaded = models.BooleanField(default=False)
 
     def __str__(self):
         return self.label
 
+    @property
+    def data(self):
+        if self.remote_loaded:
+            return self.text
+        elif self.remote_url:
+            self.text = requests.get(self.remote_url).text
+
+        self.remote_loaded = True
+        self.save()
+        return self.text
```

Comment:
New `PemData.data` property:

* Returns `text` if `remote_loaded` is `True`
* Sets `text` to the content from `remote_url` if there is one
* Sets `remote_loaded` to `True`
* Returns `text`

So basically the content is always (eventually) in `text`, with maybe a one-time load from `remote_url`.